### DISCUSSION TITLE
Feature suggestion from researcher: tables!

### DISCUSSION THREAD

**Discussion by tylerch9999** [flxzt/rnote#1216] [Ideas]
I checked the documentation and discussions and it seems like this hasn't been mentioned yet, but a quick drag-and-drop table feature would be really nice! I work in a research lab all day and I have to work with a lot of tabular data (cell counts, embryo counts, dilution calculations, reaction mixtures, etc), so it would be nice to have a feature like in GoogleDocs where blank tables of various size can be created or previous .csv and .tsv tables can be imported, viewed, and modified. Would this be hard to implement?

**Comment by Doublonmousse** [comment]
Honesty, I think it's not likely to come soon. If considered, it's ranked lower than a lot of other issues that are more pen-related (at least for now).

I don't think it's a minor addition (in terms of difficulty and amount of work).

Do you want to use rnote more like onenote (put a bunch of documents in, that can be edited alongside some additional handwritten parts ?)

**Reply by tylerch9999** [reply]
Ah, I see. I didn't have a good estimate on how hard it would be to implement. In answer to your question, I find that I don't use most of OneNote's features, but tables are an exception. I think what I'd really like to use rnote for is more like a WYSIWYG personal notebook, where I can insert pictures, tables, text, and handwritten figures/drawings.  I'd place something like 25% importance each on all of the previous media types.
